### PR TITLE
Philip/connect wallet

### PR DESCRIPTION
**Pull request by philipstanislaus**

### CODE REVIEW

File: gui/src/organisms/PlaceOrder.tsx [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-import { LimitOrderForm } from './LimitOrderForm'
+import { LimitOrderFormContainer } from './LimitOrderForm/Container'
 
 export const PlaceOrder = () => (
   <>
     <Tabs><Tab><H2>Limit</H2></Tab></Tabs>
     <Row>
       <Col>
-        <LimitOrderForm buy={true} />
+        <LimitOrderFormContainer amountDenom="XMR" priceDenom="RUNE" buy={true} />
```
Comment (haneefghanim) [inline gui/src/organisms/PlaceOrder.tsx]: Will we be allowing for orders between things other than XMR and RUNE?

File: gui/src/store/createStore.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { destroy } from 'mobx-state-tree'
-import { connectReduxDevtools } from 'mst-middlewares'
+// import { connectReduxDevtools } from 'mst-middlewares'
```
Comment (haneefghanim) [inline gui/src/store/createStore.ts]: If you merge master back into this branch, that'll take care of this.

File: gui/src/organisms/PlaceOrder.tsx (hunk 1 above)
Comment (philipstanislaus) [reply gui/src/organisms/PlaceOrder.tsx]: @haneefghanim fixed, please have another look

File: gui/src/store/createStore.ts (hunk 2 above)
Comment (philipstanislaus) [reply gui/src/store/createStore.ts]: 👍

**Comment by philipstanislaus** [comment]
@haneefghanim added changes, please have another look. In case you are not aware of that: You can see the new changes only by Shift-Clicking on a subset of commits in the "Files changed" tab like shown here: 

bildschirmfoto 2018-12-10 um 00 33 18

File: gui/src/store/Store.ts [hunk 3]
```diff
@@ -162,18 +184,30 @@ const Wallet = types.model({
...
   thorchainClientLoaded: types.boolean,
+  ui: UI,
   wallet: types.maybeNull(Wallet),
 })
 .actions(self => ({
   selectPair(pair: IPair) {
     self.pairSelected = cast(pair.id)
   },
-  createWallet: flow(function* fetchPrices() {
+  createWallet: flow(function* createWallet() {
```
Comment (haneefghanim) [inline gui/src/store/Store.ts]: Good catch

**Pull request merged**